uxarray/helpers.py

```diff
@@ -673,3 +673,49 @@ def close_face_nodes(Mesh2_face_nodes, nMesh2_face, nMaxMesh2_face_nodes):
     np.put(closed.ravel(), first_fv_idx_1d, first_node_value)
 
     return closed
+
+
+def _convert_face_node_conn_to_sparse_matrix(dense_matrix: np.ndarray) -> tuple:
```

Comment:
This function looks good now! I'd consider renaming it to something shorter since the name is quite long.

Maybe something like: `face_nodes_to_sparse_matrix`?